sdks/java/extensions/sql/pom.xml

```diff
@@ -396,6 +396,12 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-io-google-cloud-platform</artifactId>
+      <scope>provided</scope>
```

Comment:
<!--new_thread; commit:a96a74cc97c889d1b604cf399fe0890f68f41c8b; resolved:1-->
Noting that I think we are going to have to find a different way to manage the various providers. For now this follows the pattern set by Kafka so that's totally fine. But we should track how to reorg the modules so we don't have to bake in everything.

Reply:
Each provider has a dependency on SQL and its respective IO module, so we need to break providers out into their own module by IO module. The SQL CLI also depends on the providers, so we need some way for it to discover providers are actually available. I created a jira: https://issues.apache.org/jira/browse/BEAM-4190